neuron/src/app/Neuron/CLI/Types.hs

```diff
@@ -127,10 +127,10 @@ commandParser defaultNotesDir today = do
               <> long "format"
               <> help "The document format of the new zettel"
       edit <- switch (long "edit" <> short 'e' <> help "Open the newly-created zettel in $EDITOR")
-      day <-
+      created <-
         option dayReader $
-          long "day"
-            <> metavar "DAY"
+          long "created"
+            <> metavar "CREATED"
             <> value today
             <> showDefault
             <> help "Zettel creation date in UTC"
```

Comment:
"Zettel creation date/time". And it should not say "UTC" anymore (since it isn't).